services/core-api/Dockerfile

```diff
@@ -13,6 +15,19 @@ RUN pip install --no-cache-dir -r requirements.txt
 
 COPY . .
 
+# Fix log path permissions
+# This is a bit hacky but other fixes require changing app code
+RUN mkdir -p /var/log/core-api
+RUN chmod -R 700 /var/log
+RUN chown -R 1000:0 /var/log
+
+# Setup user to represent developer permissions in container
+ARG USERNAME=python
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
```

Comment:
is there a reason that these two args are being assigned like this?

Reply:
Updated: explicitly set USER_GID to 1000